client/share_trader/js/components/asset_matrix.js

```diff
@@ -8,6 +8,10 @@ import AssetStore from '../../../lib/js/react/stores/asset_store';
 const AssetMatrix = React.createClass({
 
     propTypes: {
+        assetList: React.PropTypes.oneOfType([
+            React.PropTypes.object,
```

Comment:
can be both, depending if you have the overview (with all users) or per user view